<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>875</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>447</width>
      <height>615</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/pult.PNG);</string>
    </property>
    <widget class="QPushButton" name="kvs1l">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>64</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="kvs2l">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>109</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="kvs3l">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>154</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="kvs4l">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>199</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="kvs5l">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>244</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="kvs1r">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>64</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="kvs2r">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>109</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="kvs3r">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>154</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="kvs4r">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>199</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="kvs5r">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>244</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="PlanPol">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>20</y>
       <width>57</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>ПЛАН
ПОЛ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="VypPol">
     <property name="geometry">
      <rect>
       <x>73</x>
       <y>20</y>
       <width>57</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>ВЫП
ПОЛ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PryamoNA">
     <property name="geometry">
      <rect>
       <x>133</x>
       <y>20</y>
       <width>57</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>ПРЯМО
НА</string>
     </property>
    </widget>
    <widget class="QPushButton" name="da19">
     <property name="geometry">
      <rect>
       <x>194</x>
       <y>20</y>
       <width>57</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>19</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SUO">
     <property name="geometry">
      <rect>
       <x>254</x>
       <y>20</y>
       <width>57</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>СУО</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BASU">
     <property name="geometry">
      <rect>
       <x>314</x>
       <y>20</y>
       <width>57</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>БАСУ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PolDann">
     <property name="geometry">
      <rect>
       <x>374</x>
       <y>20</y>
       <width>57</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>ПОЛ
ДАНН</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Maneur">
     <property name="geometry">
      <rect>
       <x>194</x>
       <y>310</y>
       <width>57</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>МА
НЕВР</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RTS">
     <property name="geometry">
      <rect>
       <x>133</x>
       <y>310</y>
       <width>57</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>РТС</string>
     </property>
    </widget>
    <widget class="QPushButton" name="KORR">
     <property name="geometry">
      <rect>
       <x>314</x>
       <y>310</y>
       <width>57</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>КОРР</string>
     </property>
    </widget>
    <widget class="QPushButton" name="NAV">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>310</y>
       <width>57</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>НАВ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ISP">
     <property name="geometry">
      <rect>
       <x>374</x>
       <y>310</y>
       <width>57</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>ИСП</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Tzad">
     <property name="geometry">
      <rect>
       <x>73</x>
       <y>310</y>
       <width>57</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>Тзад</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ArvArp">
     <property name="geometry">
      <rect>
       <x>254</x>
       <y>310</y>
       <width>57</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>АРВ
АРП</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UprMFPU">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>496</y>
       <width>50</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>УПР
МФПУ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="InfBKS">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>496</y>
       <width>50</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>ИНФ
БКС</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ENG">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>496</y>
       <width>50</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>ENG</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CIFR">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>556</y>
       <width>50</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>ЦИФР</string>
     </property>
    </widget>
    <widget class="QPushButton" name="empty">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>556</y>
       <width>50</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="IIPE">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>556</y>
       <width>50</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>ИИПЭ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="q">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>350</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Q
     Й</string>
     </property>
    </widget>
    <widget class="QPushButton" name="a">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>390</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>A
     Ф</string>
     </property>
    </widget>
    <widget class="QPushButton" name="z">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>430</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Z
     Я</string>
     </property>
    </widget>
    <widget class="QPushButton" name="x">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>430</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>X
     Ч</string>
     </property>
    </widget>
    <widget class="QPushButton" name="w">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>350</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>W
     Ц</string>
     </property>
    </widget>
    <widget class="QPushButton" name="s">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>390</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>S
     Ы</string>
     </property>
    </widget>
    <widget class="QPushButton" name="e">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>350</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>E
     У</string>
     </property>
    </widget>
    <widget class="QPushButton" name="d">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>390</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>D
     В</string>
     </property>
    </widget>
    <widget class="QPushButton" name="c">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>430</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>C
     С</string>
     </property>
    </widget>
    <widget class="QPushButton" name="v">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>430</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>V
     М</string>
     </property>
    </widget>
    <widget class="QPushButton" name="y">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>350</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Y
     Н</string>
     </property>
    </widget>
    <widget class="QPushButton" name="r">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>350</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>R
     К</string>
     </property>
    </widget>
    <widget class="QPushButton" name="t">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>350</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>T
     Е</string>
     </property>
    </widget>
    <widget class="QPushButton" name="g">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>390</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>G
     П</string>
     </property>
    </widget>
    <widget class="QPushButton" name="n">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>430</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>N
     Т</string>
     </property>
    </widget>
    <widget class="QPushButton" name="f">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>390</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>F
     А</string>
     </property>
    </widget>
    <widget class="QPushButton" name="h">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>390</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>H
     Р</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>430</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>B
     И</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ruG">
     <property name="geometry">
      <rect>
       <x>365</x>
       <y>390</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>
     Ж</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m">
     <property name="geometry">
      <rect>
       <x>245</x>
       <y>430</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>M
     Ь</string>
     </property>
    </widget>
    <widget class="QPushButton" name="o">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>350</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>O
     Щ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="p">
     <property name="geometry">
      <rect>
       <x>365</x>
       <y>350</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>P
     З</string>
     </property>
    </widget>
    <widget class="QPushButton" name="u">
     <property name="geometry">
      <rect>
       <x>245</x>
       <y>350</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>U
     Г</string>
     </property>
    </widget>
    <widget class="QPushButton" name="i">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>350</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>I
     Ш</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ruA">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>390</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>
     Э</string>
     </property>
    </widget>
    <widget class="QPushButton" name="vvd">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>430</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>ВВД</string>
     </property>
    </widget>
    <widget class="QPushButton" name="k">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>390</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>K
     Л</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ruTznak">
     <property name="geometry">
      <rect>
       <x>365</x>
       <y>430</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>/
     Ъ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ruU">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>430</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>.
     Ю</string>
     </property>
    </widget>
    <widget class="QPushButton" name="j">
     <property name="geometry">
      <rect>
       <x>245</x>
       <y>390</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>J
     О</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ruX">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>350</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string> 
     Х</string>
     </property>
    </widget>
    <widget class="QPushButton" name="l">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>390</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>L
     Д</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ruB">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>430</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>,
     Б</string>
     </property>
    </widget>
    <widget class="QPushButton" name="three">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>474</y>
       <width>37</width>
       <height>37</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="seven">
     <property name="geometry">
      <rect>
       <x>245</x>
       <y>554</y>
       <width>37</width>
       <height>37</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zero">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>474</y>
       <width>37</width>
       <height>37</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="six">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>514</y>
       <width>37</width>
       <height>37</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="four">
     <property name="geometry">
      <rect>
       <x>245</x>
       <y>514</y>
       <width>37</width>
       <height>37</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PlusMinus">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>514</y>
       <width>37</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>±</string>
     </property>
    </widget>
    <widget class="QPushButton" name="five">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>514</y>
       <width>37</width>
       <height>37</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eight">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>554</y>
       <width>37</width>
       <height>37</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="two">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>474</y>
       <width>37</width>
       <height>37</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nine">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>554</y>
       <width>37</width>
       <height>37</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="_">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>554</y>
       <width>37</width>
       <height>37</height>
      </rect>
     </property>
     <property name="text">
      <string>_</string>
     </property>
    </widget>
    <widget class="QPushButton" name="one">
     <property name="geometry">
      <rect>
       <x>245</x>
       <y>474</y>
       <width>37</width>
       <height>37</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>477</y>
       <width>57</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>СТИР</string>
     </property>
    </widget>
    <widget class="QPushButton" name="menu">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>517</y>
       <width>57</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>МЕНЮ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sbros">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>557</y>
       <width>36</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>СБР</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>341</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#004c88;&quot;&gt;МЕНЮ&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#004c88;&quot;&gt;&amp;lt;БАЗЫ                                                                               ИНС&amp;gt;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt; font-weight:600; color:#004c88;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#004c88;&quot;&gt;&amp;lt;ИШР				        СНС&amp;gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>934</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
